src/Analyzers/CSharp/Analysis/UseConditionalAccessAnalyzer.cs

```diff
@@ -274,6 +274,14 @@ static bool ExistsImplicitConversionToBoolean(INamedTypeSymbol typeSymbol)
                 case SyntaxKind.GreaterThanOrEqualExpression:
                 case SyntaxKind.EqualsExpression:
                     {
+                        var leftTypeSymbol = semanticModel.GetTypeSymbol(((BinaryExpressionSyntax)expression).Left);
+                        if (leftTypeSymbol.IsValueType && leftTypeSymbol.SpecialType is SpecialType.None)
```

Comment:
I would rather use `CSharpFacts.IsPredefinedType` instead of `is SpecialTypen.None`.